### ISSUE TITLE
Customer information page - Terminate / Reactivate Buttons

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#1433]
In GitLab by @kdjstudios on Apr 8, 2019, 08:20

**Submitted by:** Kyle

**Helpdesk:** NA

**Server:** All

**Client/Site:** ALL 

**Account:** ALL

**Issue:** 

**Customer Page**

Also The delete/terminate button is shown for accounts that are already terminated? This should probably be a separate ticket as it does not directly relate to SD.

- Yeah, this can be a separate ticket. We suggest to add the re-activate button for the terminated accounts.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Apr 8, 2019, 08:21

Created new ticket from comment in #28. We will want to have on the customer information screen the terminate buttons turn into reactivate buttons for already terminated accounts. I marked this as 2 priority as no one has complained about this yet. Please provide an estimate.

**Comment by kdjstudios** [comment]
In GitLab by @amishra on May 23, 2019, 04:07

@kdjstudios 


Estimate:

|Size|Lower Time Bound|Variability|Upper Time Bound|Testing|Time for Testing| Time for Dev & Testing|
|-|-|-|-|-|-|-|
|X-Small|1|Worked with Concepts, implementation ok|1.5|Low Impact|0.5|2|

Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @amishra on May 23, 2019, 06:21

assigned to @amishra

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 5, 2019, 10:01

@amishra 

This is approved.

**Comment by kdjstudios** [comment]
In GitLab by @amishra on Jun 10, 2019, 07:03

@kdjstudios 


Thank you for your approval.

We have completed the development as you requested. Please test it on the merge2 server.


Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 10, 2019, 09:00

@amishra 

Thank you. This looks great and is working as expected. marking for announcement.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Jun 11, 2019, 05:30

changed time estimate to 2h

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Jun 12, 2019, 07:32

@kdjstudios 

While testing we noticed one thing which we would like to share with you:

* There is a blurred memory in my head that there is a ticket which has mentioned that we need to track the Turn On/Off date which can be useful on many of the support tickets in near future, as currently, we are not tracking the record of those dates.

* As currently, a user can terminate/reactivate an account multiple times & the dates may have an effect on many cases in the billing.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 12, 2019, 09:44

@pchaudhary @tim.traylor 

I think #1378 is addressing those concerns. I believe we are still waiting on that ticket to find out all the places that are effected by the dates. 

I also do recall on a few other tickets (or maybe it was just on a call) it being mentioned that we may want to clear those dates during a reactivation possibly. I can not find those tickets currently.

**Comment by kdjstudios** [comment]
In GitLab by @amishra on Jun 22, 2019, 03:48

@kdjstudios 


We have successfully released on both of the production servers.

Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 22, 2019, 13:22

@amishra 

This works as expected. Marking for close

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jun 24, 2019, 14:27

closed
